zsales_class_git.clas.abap

```diff
@@ -12,11 +12,14 @@ CLASS zsales_class_git DEFINITION
         matnr TYPE matnr,
         charg TYPE charg_d,
         werks TYPE werks_d,
-        matkl TYPE matkl,
+        lgort TYPE lgort_d,  "New Change V4 - Branch 5
```

Comment:
Specify table name in structure

Reply:
The above review comment has been fixed.. Please review again